app/controllers/SubmissionsController.scala

```diff
@@ -59,7 +60,8 @@ class SubmissionsController @Inject()(
     customsDeclareExportsConnector.findSubmission(id).flatMap {
       case Some(submission) =>
         customsDeclareExportsConnector.findNotifications(id).map { notifications =>
-          Ok(declarationInformationPage(submission, notifications))
+          val hasDmsdocNotifications = notifications.exists(_.status == SubmissionStatus.ADDITIONAL_DOCUMENTS_REQUIRED)
+          Ok(declarationInformationPage(submission, notifications, hasDmsdocNotifications))
```

Comment:
This logic seems rather simple. Wouldn't it be worth having this inside `declaration_information` template instead of an extra flag parameter?
From testing perspective, I don't think it makes much of a difference too.

Reply:
Also, this logic is related strictly to what will be present on the page, so maybe controller should not really care about such details?

Reply:
NP, since you already follow this approach project-wide. I did this way because I generally prefer the idea "no logic in templates".